### DISCUSSION TITLE
Find feature in SEB Mac

### DISCUSSION THREAD

**Discussion by uskomatonta** [SafeExamBrowser/seb-mac#161] [Q&A]
Hello, in iOS SEB it is possible to use Find feature command F. But not possible on Mac. When it could be implemented? Or workarounds?

Thanks!

**Comment by danschlet** [comment]
It is planed for SEB 3.1, most likely to be released in January.

**Reply by prashal532** [reply]
Hello is there a delay in SEB MAC 3.1 release
